app/models/lookup_key.rb

```diff
@@ -201,6 +205,7 @@ def array2path(array)
 
   def validate_and_cast_default_value
     return true if default_value.nil?
+    return true if contains_erb?(default_value)
```

Comment:
Just add this if to the previous statement as in `default_value.nil? || contains_erb?(default_value)`